docker-compose/docker-compose.base.yml

```diff
@@ -13,12 +13,41 @@ services:
       - 9200:9200
       - 9300:9300
     environment:
-      -  discovery.type=single-node
+      - discovery.type=single-node
+      - LOGSPOUT=ignore
 
   kibana:
     image: docker.elastic.co/kibana/kibana:6.8.6
     ports:
       - 5601:5601
+    environment:
+      - LOGSPOUT=ignore
+    depends_on:
+      - elasticsearch
+
+  logstash:
+    image: docker.elastic.co/logstash/logstash:6.8.6
+    volumes:
+      - $PWD/../pipelines.yml:/config/pipelines.yml:Z
+      - $PWD/../logstash.conf:/usr/share/logstash/pipeline/logstash.conf:Z
+    environment:
+      - LOGSPOUT=ignore
+    depends_on:
+      - elasticsearch
+      - kibana
+
+  logspout:
+    image: gliderlabs/logspout:v3
+    command: 'udp://logstash:5000'
+    environment:
+      - LOGSPOUT=ignore
+      - filter.name=gridsuite*
```

Comment:
The name is either gridsuite or gridmerge or gridstudy.
And we can have all those environement at the same time, and logspout should only collect the logs for its own env. Can you make it work with env variables ? Something like filter.name=${NAME} (I don't know the exact syntax)

Reply:
For now we can't have different environments at the same time because of hardcoded ports on the host, so let's just keep grid* for simplicity